### PR TITLE
Add simple move and rotate

### PR DESCRIPTION
**Pull request by imadik**

### CODE REVIEW

File: src/Rotate/Rotate.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "Interfaces/IRotable.h"
+
+namespace SpaceBattle
+{
+    class Rotate
```
Comment (a-poliakov) [inline src/Rotate/Rotate.h]: let's extend abstract class Command

File: src/Move/Move.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "Interfaces/IMovable.h"
+
+namespace SpaceBattle
+{
+    class Move
```
Comment (a-poliakov) [inline src/Move/Move.h]: let's extend abstract class Command

File: src/Interfaces/IMovable.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "Types/Types.h"
+
+namespace SpaceBattle
```
Comment (a-poliakov) [inline src/Interfaces/IMovable.h:5]: let's create abstract class Command:
```
class Command {
 void execute();
}
```

**Review by a-poliakov — APPROVED** [review]
lgtm

File: src/Rotate/Rotate.h (hunk 1 above)
Comment (imadik) [reply src/Rotate/Rotate.h]: done

File: src/Move/Move.h (hunk 2 above)
Comment (imadik) [reply src/Move/Move.h]: done

File: src/Interfaces/IMovable.h (hunk 3 above)
Comment (imadik) [reply src/Interfaces/IMovable.h:5]: done

**Pull request merged**